lib/bin_conf/Bin_conf.ml

```diff
@@ -207,6 +211,15 @@ let name =
     ~set:(fun name conf -> {conf with name})
     Arg.(value & opt (some string) default & info ["name"] ~doc ~docs ~docv)
 
+let on_save =
+  let doc =
+    "Make OCamlformat not format if no .ocamlformat file could be found. \
+     For code editors."
```

Comment:
It would be nice to mention "project root" and the way the `.ocamlformat` files are searched in some ways.

And mention the intent: (I'm not sure about the wording, please edit)
```suggestion
     Avoid formatting files that shouldn't be when used by text editors to format on each save."
```